tests/pyfunc/test_spark.py

```diff
@@ -105,9 +107,9 @@ def get_spark_session(conf):
     # adding `.config("spark.driver.bindAddress", "127.0.0.1")` to the SparkSession
     # builder configuration will enable a SparkSession to start.
     # For local testing, uncomment the following line:
-    # spark_master = os.environ.get("SPARK_MASTER", "local[1]")
+    spark_master = os.environ.get("SPARK_MASTER", "local[1]")
     # If doing local testing, comment-out the following line.
-    spark_master = os.environ.get("SPARK_MASTER", "local-cluster[2, 1, 1024]")
+    # spark_master = os.environ.get("SPARK_MASTER", "local-cluster[2, 1, 1024]")
```

Comment:
Do we need this change?

Reply:
Ah I forgot to change it back for local testing! I'll change it back